1.9/metrics/datadog.md

```diff
@@ -0,0 +1,137 @@
+---
+post_title: Datadog Metrics for DC/OS
+feature_maturity: preview
+menu_order: 3.5
+---
+
+The Datadog metrics plugin for DC/OS supports sending metrics from the DC/OS metrics service on master and agent hosts to a Datadog agent for shipping to DatadogHQ.
+
+# Installation
+
+## Build the plugin
+
+**Prerequisite:** 
+- DC/OS is installed
+- [Go programming environment](https://golang.org/doc/install) <!-- dcos-metrics must be run from within the go directory -->
+-  Git:
+   -  **macOS:** Get the installer from [Git downloads](http://git-scm.com/download/mac).
+   -  **Unix/Linux:** See these <a href="https://git-scm.com/book/en/v2/Getting-Started-Installing-Git" target="_blank">installation instructions</a>.
+
+1. Clone the DC/OS metrics repository:
+
+   ```bash
+   $ git clone git@github.com:dcos/dcos-metrics
+   ```
+   
+1. Navigate to the `dcos-metrics` repository and run the build command:
+
+   ```bash
+   $ cd dcos-metrics && make && make plugins
+   ```
+
+   The plugin is available in the build directory:
+
+   ```
+   $ tree build
+   build
+   ├── collector
+   │   └── dcos-metrics-collector-1.0.0-rc7
+   ├── plugins
+   │   └── dcos-metrics-datadog_plugin-1.0.0-rc7
+   └── statsd-emitter
+       └── dcos-metrics-statsd-emitter-1.0.0-rc7
+   ```
+
+## Install the Datadog agent in your DC/OS cluster
+
+Install the `datadog` package in DC/OS:
+
+1.  Go to the **Universe** tab of the DC/OS GUI and find the **Datadog** package. 
+    ![datadog package](/docs/1.9/img/datadog-package.png)
+1.  Click **INSTALL PACKAGE** -> **ADVANCED INSTALLATION** and enter [your Datadog API_KEY](https://app.datadoghq.com/account/settings#api).
+1.  Click **REVIEW AND INSTALL** to complete your installation.
+
+After a few minutes, a Datadog agent will be running in the cluster at `datadog-agent.marathon.mesos:8125`. This is the default location used by the Datadog plugin.
+
+## Test the DC/OS Datadog metrics plugin (agents only)
```

Comment:
@philipnrmn is the rest of the information (lines 57-137) all testing, or is testing just lines 57-70? Please clarify.

CC: @sascala